src/SetDocAttrStep.js

```diff
@@ -48,7 +48,7 @@ class SetDocAttrStep extends Step {
     };
   }
 
-  static fromJSON(json: SetDocAttrStepJSONValue) {
+  static fromJSON(schema: Schema, json: SetDocAttrStepJSONValue) {
```

Comment:
@alanhoff :

This does not seem match what the [document says](http://prosemirror.net/docs/ref/#state.EditorState^fromJSON).

Could you please try this instead?

```
static fromJSON(config: {schema: Schema}, json: SetDocAttrStepJSONValue) {
```

Reply:
`EditorState.fromJSON` is for deserializing a JSON representation of an `EditorState` but `SetDocAttrStep` is a `Step`, not an `EditorState`, the correct method signature is [here][0].

@hedgerwang replying to your previous question: I found this issue while trying to implement [`prosemirror-collab`][1]. For applying a remote `Step` into the local `EditorState` we first need to deserialize the inbound step, as demonstrated [here][2].  


[0]: http://prosemirror.net/docs/ref/#transform.Step%5EfromJSON
[1]: https://github.com/ProseMirror/prosemirror-collab
[2]: https://github.com/ProseMirror/website/blob/master/src/collab/client/collab.js#L131